dark1/memberavatarstatus/event/listener.php

```diff
@@ -77,26 +77,31 @@ static public function getSubscribedEvents()
 			// Main Setup
```

Comment:
Too much of events in only one listener.
All this should be divided into localized listeners by purpose.

Reply:
Done 👍